content/_index.md

```diff
@@ -1,9 +1,10 @@
 ---
-heading: Program the cloud.
+heading: Modern Infrastructure as Code.
+
 
 subhead: >
-    **Modern infrastructure as code**. Create, deploy, and manage infrastructure on
-    any cloud, using your favorite language.
+    Use your favorite languages and tools. Empower your developers
```

Comment:
See I'd really question if we should add this - I mean, what if my favourite language is C# / F#?

Wouldn't it be better to say something like, "Use real programming languages and tooling to empower your developers and operators to work better together" ?

Reply:
I second this. We should not say “your favorite language” until we support any language.